### ISSUE TITLE
Add travis integration

### ISSUE THREAD

**Issue by codfish** [codfish/jquery-data-remote#2]
I need to find a way to build production ready assets, e.g. minified version of the plugin, without needing to include those in git for tracking. Travis may be able to work for this...

[ ] Attempt to add build task for npm. http://docs.travis-ci.com/user/deployment/npm/

**Comment by codonnell822** [comment]
> find a way to build production ready assets

after a bunch of incorrect google searches on what I needed, I realized it was pretty damn simple. All I needed to do was to add an `.npmignore` file, so that the `dist/` (where minified assets are built) directory was not ignored. I had that directory ignored by `.gitignore` and apparently npm will use that file as it's ignore file if you don't explicitly provide one:

https://docs.npmjs.com/misc/developers

> Use a .npmignore file to keep stuff out of your package. If there's no .npmignore file, but there is a .gitignore file, then npm will ignore the stuff matched by the .gitignore file. If you want to include something that is excluded by your .gitignore file, you can create an empty .npmignore file to override it."

Keeping this issue open however because I would still like to set up Travis integration in order to automatically publish the plugin to npm for me, everytime there's a new version.
